Wwise SDK setup blocks Pre-built SDK O3DE build

**Describe the bug**
If a Wwise SDK is present on the machine and engine configuration picks it up, the Pre-built SDK build will fail. It should be noted that on the Source configuration the build goes through with the same Wwise SDK present.

Part of the Command Line log of building with failures (for full log please refer to the **Command Line log** section):
```
  DccScriptingInterface.Editor.vcxproj -> C:\o3de\build\windows_vs2019\bin\profile\DccScriptingInterface.Editor.dll
C:\o3de\Gems\AudioEngineWwise\Code\Source\AudioEngineWwiseGemSystemComponent.cpp(124,40): error C2039: 'Descriptor': is not a member of 'Audio::AudioImplAllocator' [C:\o3de\build\windows_vs2019\External\AudioEngineWwise-b
ec3ca47\Code\AudioEngineWwise.Static.vcxproj]
              Audio::AudioImplAllocator::Descriptor allocDesc;
                                         ^
  C:\o3de\Gems\AudioSystem\Code\Include\Engine\AudioAllocators.h(29): note: see declaration of 'Audio::AudioImplAllocator'
      class AudioImplAllocator final
C:\o3de\Gems\AudioEngineWwise\Code\Source\AudioEngineWwiseGemSystemComponent.cpp(124,51): error C2065: 'Descriptor': undeclared identifier [C:\o3de\build\windows_vs2019\External\AudioEngineWwise-bec3ca47\Code\AudioEngineW
wise.Static.vcxproj]
              Audio::AudioImplAllocator::Descriptor allocDesc;
                                                    ^
C:\o3de\Gems\AudioEngineWwise\Code\Source\AudioEngineWwiseGemSystemComponent.cpp(124,51): error C2146: syntax error: missing ';' before identifier 'allocDesc' [C:\o3de\build\windows_vs2019\External\AudioEngineWwise-bec3ca
47\Code\AudioEngineWwise.Static.vcxproj]
              Audio::AudioImplAllocator::Descriptor allocDesc;
                                                    ^
C:\o3de\Gems\AudioEngineWwise\Code\Source\AudioEngineWwiseGemSystemComponent.cpp(124,51): error C2065: 'allocDesc': undeclared identifier [C:\o3de\build\windows_vs2019\External\AudioEngineWwise-bec3ca47\Code\AudioEngineWw
ise.Static.vcxproj]
              Audio::AudioImplAllocator::Descriptor allocDesc;
                                                    ^
C:\o3de\Gems\AudioEngineWwise\Code\Source\AudioEngineWwiseGemSystemComponent.cpp(127,13): error C2065: 'allocDesc': undeclared identifier [C:\o3de\build\windows_vs2019\External\AudioEngineWwise-bec3ca47\Code\AudioEngineWw
ise.Static.vcxproj]
              allocDesc.m_allocationRecords = true;
              ^
C:\o3de\Gems\AudioEngineWwise\Code\Source\AudioEngineWwiseGemSystemComponent.cpp(128,13): error C2065: 'allocDesc': undeclared identifier [C:\o3de\build\windows_vs2019\External\AudioEngineWwise-bec3ca47\Code\AudioEngineWw
ise.Static.vcxproj]
              allocDesc.m_heap.m_numFixedMemoryBlocks = 1;
              ^
C:\o3de\Gems\AudioEngineWwise\Code\Source\AudioEngineWwiseGemSystemComponent.cpp(129,13): error C2065: 'allocDesc': undeclared identifier [C:\o3de\build\windows_vs2019\External\AudioEngineWwise-bec3ca47\Code\AudioEngineWw
ise.Static.vcxproj]
              allocDesc.m_heap.m_fixedMemoryBlocksByteSize[0] = poolSize;
              ^
C:\o3de\Gems\AudioEngineWwise\Code\Source\AudioEngineWwiseGemSystemComponent.cpp(131,13): error C2065: 'allocDesc': undeclared identifier [C:\o3de\build\windows_vs2019\External\AudioEngineWwise-bec3ca47\Code\AudioEngineWw
ise.Static.vcxproj]
              allocDesc.m_heap.m_fixedMemoryBlocks[0] = AZ::AllocatorInstance<AZ::OSAllocator>::Get().Allocate(
              ^
C:\o3de\Gems\AudioEngineWwise\Code\Source\AudioEngineWwiseGemSystemComponent.cpp(132,17): error C2065: 'allocDesc': undeclared identifier [C:\o3de\build\windows_vs2019\External\AudioEngineWwise-bec3ca47\Code\AudioEngineWw
ise.Static.vcxproj]
                  allocDesc.m_heap.m_fixedMemoryBlocksByteSize[0], allocDesc.m_heap.m_memoryBlockAlignment);
                  ^
C:\o3de\Gems\AudioEngineWwise\Code\Source\AudioEngineWwiseGemSystemComponent.cpp(132,66): error C2065: 'allocDesc': undeclared identifier [C:\o3de\build\windows_vs2019\External\AudioEngineWwise-bec3ca47\Code\AudioEngineWw
ise.Static.vcxproj]
                  allocDesc.m_heap.m_fixedMemoryBlocksByteSize[0], allocDesc.m_heap.m_memoryBlockAlignment);
                                                                   ^
  RecastNavigation.Editor.Tests.vcxproj -> C:\o3de\build\windows_vs2019\bin\profile\RecastNavigation.Editor.Tests.dll
C:\o3de\Gems\AudioEngineWwise\Code\Source\AudioEngineWwiseGemSystemComponent.cpp(134,70): error C2065: 'allocDesc': undeclared identifier [C:\o3de\build\windows_vs2019\External\AudioEngineWwise-bec3ca47\Code\AudioEngineWw
ise.Static.vcxproj]
              AZ::AllocatorInstance<Audio::AudioImplAllocator>::Create(allocDesc);
                                                                       ^
```

**Prerequisites**
* Wwise SDK installed (verified with version 2021.1.7601).

**Steps to reproduce**
1. Follow Pre-built SDK configuration build instructions from the https://www.o3de.org/docs/welcome-guide/setup/setup-from-github/building-windows/ documentation page.

**Expected behavior**
Engine is built.

**Actual behavior**
Build fails, o3de-install folder is not created.

**Command Line log**
[WwiseLog.txt](https://github.com/o3de/o3de/files/10021236/WwiseLog.txt)

**Found in Branch**
Development (5234076)

**Desktop**
 - Device: PC
 - OS: Windows
 - Version 11
 - CPU AMD Ryzen 5 3600
 - GPU NVIDIA GeForce RTX 2060 SUPER
 - Memory 16GB

--- a/Gems/AudioEngineWwise/Code/Source/AudioEngineWwiseGemSystemComponent.cpp
+++ b/Gems/AudioEngineWwise/Code/Source/AudioEngineWwiseGemSystemComponent.cpp
@@ -10,7 +10,6 @@
 
 #include <AzCore/PlatformDef.h>
 #include <AzCore/Console/ILogger.h>
-#include <AzCore/Memory/OSAllocator.h>
 #include <AzCore/Serialization/SerializeContext.h>
 #include <AzCore/Serialization/EditContext.h>
 #include <AzCore/Serialization/EditContextConstants.inl>
@@ -116,30 +115,15 @@ namespace AudioEngineWwiseGem
         AZ_Assert(Audio::Wwise::Cvars::s_PrimaryMemorySize > memorySubpartitionSizes,
             "Wwise memory sizes of sub-categories add up to more than the primary memory pool size!")
 
-        // Initialize memory block for Wwise to use...
-        if (!AZ::AllocatorInstance<Audio::AudioImplAllocator>::IsReady())
-        {
-            const size_t poolSize = Audio::Wwise::Cvars::s_PrimaryMemorySize << 10;
-
-            Audio::AudioImplAllocator::Descriptor allocDesc;
-
-            // Generic Allocator:
-            allocDesc.m_allocationRecords = true;
-            allocDesc.m_heap.m_numFixedMemoryBlocks = 1;
-            allocDesc.m_heap.m_fixedMemoryBlocksByteSize[0] = poolSize;
-
-            allocDesc.m_heap.m_fixedMemoryBlocks[0] = AZ::AllocatorInstance<AZ::OSAllocator>::Get().Allocate(
-                allocDesc.m_heap.m_fixedMemoryBlocksByteSize[0], allocDesc.m_heap.m_memoryBlockAlignment);
-
-            AZ::AllocatorInstance<Audio::AudioImplAllocator>::Create(allocDesc);
-        }
-
         AZ::SettingsRegistryInterface::FixedValueString assetPlatform = AzFramework::OSPlatformToDefaultAssetPlatform(
             AZ_TRAIT_OS_PLATFORM_CODENAME);
-        if (auto settingsRegistry = AZ::SettingsRegistry::Get(); settingsRegistry != nullptr)
+        if (assetPlatform.empty())
         {
-            AZ::SettingsRegistryMergeUtils::PlatformGet(*settingsRegistry, assetPlatform,
-                AZ::SettingsRegistryMergeUtils::BootstrapSettingsRootKey, "assets");
+            if (const auto settingsRegistry = AZ::SettingsRegistry::Get(); settingsRegistry != nullptr)
+            {
+                AZ::SettingsRegistryMergeUtils::PlatformGet(*settingsRegistry, assetPlatform,
+                    AZ::SettingsRegistryMergeUtils::BootstrapSettingsRootKey, "assets");
+            }
         }
 
         m_engineWwise = AZStd::make_unique<Audio::CAudioSystemImpl_wwise>(assetPlatform.c_str());
@@ -163,11 +147,6 @@ namespace AudioEngineWwiseGem
     void AudioEngineWwiseGemSystemComponent::Release()
     {
         m_engineWwise.reset();
-
-        if (AZ::AllocatorInstance<Audio::AudioImplAllocator>::IsReady())
-        {
-            AZ::AllocatorInstance<Audio::AudioImplAllocator>::Destroy();
-        }
     }
 
 #if defined(AUDIO_ENGINE_WWISE_EDITOR)

--- a/Gems/AudioEngineWwise/Code/Tests/AudioEngineWwiseTest.cpp
+++ b/Gems/AudioEngineWwise/Code/Tests/AudioEngineWwiseTest.cpp
@@ -54,46 +54,18 @@ namespace UnitTest
 
         void SetupEnvironment() override
         {
-            // Create AZ::SystemAllocator
-            if (!AZ::AllocatorInstance<AZ::SystemAllocator>::IsReady())
-            {
-                AZ::AllocatorInstance<AZ::SystemAllocator>::Create();
-            }
-
             // Setup Mocks on a stub environment
             m_mocks = new MockHolder();
 
             m_stubEnv.pConsole = &m_mocks->m_console;
             m_stubEnv.pSystem = &m_mocks->m_system;
             gEnv = &m_stubEnv;
-
-            // Create AudioImplAllocator (used by Wwise)
-            if (!AZ::AllocatorInstance<AudioImplAllocator>::IsReady())
-            {
-                AudioImplAllocator::Descriptor allocDesc;
-                allocDesc.m_allocationRecords = false;
-                allocDesc.m_heap.m_numFixedMemoryBlocks = 1;
-                allocDesc.m_heap.m_fixedMemoryBlocksByteSize[0] = 8 << 20;
-                allocDesc.m_heap.m_fixedMemoryBlocks[0] = AZ::AllocatorInstance<AZ::OSAllocator>::Get().Allocate(allocDesc.m_heap.m_fixedMemoryBlocksByteSize[0], allocDesc.m_heap.m_memoryBlockAlignment);
-
-                AZ::AllocatorInstance<AudioImplAllocator>::Create(allocDesc);
-            }
         }
 
         void TeardownEnvironment() override
         {
-            if (AZ::AllocatorInstance<AudioImplAllocator>::IsReady())
-            {
-                AZ::AllocatorInstance<AudioImplAllocator>::Destroy();
-            }
-
             delete m_mocks;
             m_mocks = nullptr;
-
-            if (AZ::AllocatorInstance<AZ::SystemAllocator>::IsReady())
-            {
-                AZ::AllocatorInstance<AZ::SystemAllocator>::Destroy();
-            }
         }
 
     private:

--- a/Gems/AudioSystem/Code/Source/AudioSystemGemSystemComponent.cpp
+++ b/Gems/AudioSystem/Code/Source/AudioSystemGemSystemComponent.cpp
@@ -11,7 +11,6 @@
 #include <AzCore/Console/IConsole.h>
 #include <AzCore/Console/ILogger.h>
 #include <AzCore/Interface/Interface.h>
-#include <AzCore/Memory/OSAllocator.h>
 #include <AzCore/Module/Environment.h>
 #include <AzCore/Serialization/SerializeContext.h>
 #include <AzCore/Serialization/EditContext.h>

